script/Arcueid/Day 6/06_00_ARC06_1B.txt

```diff
@@ -76,7 +76,7 @@ She must have hurried to fetch it during the small time window while I was chang
 -- 「ありがとう。正直、歩くたびにカカトから飛び上がりそうだったんだ。すごく助かる」
 // This is pretty liberal, but I'm not sure of a good direct idiom for
 // 'カカトから飛び上がりそうだった' and a literal TL just doesn't make sense
-"Thank you. To be honest, I felt as if my foot was about to fall off with every step. This is a lifesaver."
+"Thank you. To be honest, I felt as if my leg was about to fall off with every step. This is a lifesaver."
```

Comment:
I feel like foot might be a bit more idiomatic here? Leg/foot distinction is a bit rough here and in one of the other files iirc

Reply:
it is indeed a bit up in the air in this file and 6_1 _but_ I went with "leg" here because 
"my leg is about to fall off" is a natural thing to say
AND
we use leg just about everywhere else. Same word -> just trying to be consistent.